tests/Feature/UsersTest.php

```diff
@@ -28,49 +28,30 @@ class UsersTest extends TestCase
 {
 	use InteractWithSmartAlbums;
 
-	public function testSetLogin(): void
+	public function testSetAminLoginIfAdminUnconfigured(): void
```

Comment:
Added this on the PR for this too, but: 
```suggestion
	public function testSetAdminLoginIfAdminUnconfigured(): void
```

Reply:
Not part of this PR anymore as this has already been merged in https://github.com/LycheeOrg/Lychee/pull/1471.